src/main/java/pl/pop/interview/master/answer/AnswerManager.java

```diff
@@ -32,12 +32,12 @@ public QuestionDTO generateRandomQuestion() {
     }
 
     @Override
-    public AnswerDTO save(Long questionId, String answer) {
+    public AnswerDTO save(Long questionId, boolean isTrue) {
```

Comment:
skoro wyżej, tj. w DTO, została zmieniona nazwa z "isCorrect" na "answer", to tutaj też może być tak nazwane. 
Wtedy będzie jaśniej informowało co z tym się dzieje.